packages/lib/utils/frontMatter.ts

```diff
@@ -187,11 +187,14 @@ export const parse = (note: string): ParsedMeta => {
 
 	const md = toLowerCase(yaml.load(header, { schema: yaml.FAILSAFE_SCHEMA }) ?? {});
 	const metadata: NoteEntity = {
-		title: md['title'] || '',
 		source_url: md['source'] || '',
 		is_todo: ('completed?' in md) ? 1 : 0,
 	};
 
+	if ('title' in md) {
+		metadata['title'] = md.title;
+	}
+
```

Comment:
This is not where the fix should be - this file is a generic one. It should parse the metadata and return an empty string for the title if none is available. The fix should be applied elsewhere, probably where the file is being imported

Reply:
Okay. Here's the other option: https://github.com/laurent22/joplin/pull/12698